### PR TITLE
DOCS-3372 Synthetic Monitoring Settings Edit

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Updates the Settings Variable documentation with more capabilities using regex.

### Motivation

DOCS-3372

### Preview

https://docs-staging.datadoghq.com/alai97/synthetics-settings-variable-regex-doc-update/synthetics/settings/?tab=createfromhttptest

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/synthetics/settings/_index.md [hunk 1]
```diff
@@ -62,7 +62,19 @@ You can create variables from your existing [HTTP tests][1] by parsing their ass
 
 {{< img src="synthetics/settings/variable_fromhttp_3.png" alt="Variable from HTTP Test" style="width:80%;">}}
 
-**Note:** Variable values are updated whenever the test they are extracted from runs.
+In addition to extracting a value with a `regex`, you can also:
```
Comment (urseberry) [inline content/en/synthetics/settings/_index.md:65]: Can you rephrase this to be a full sentence? Full sentences are easier for the translators to localize. I attempted to rephrase this, but I'm not confident of the context and meaning
```suggestion
In addition to extracting a value with a regex, you can also use a regex to parse the following examples:
```

Comment (peculiarism) [inline content/en/synthetics/settings/_index.md:65]: I think we should link to the MDN documentation for regex patterns: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions

File: content/en/synthetics/settings/_index.md [hunk 2]
```diff
@@ -62,7 +62,19 @@ You can create variables from your existing [HTTP tests][1] by parsing their ass
...
+
+  - Match not only the first instance of a pattern, but also all instances of the supplied pattern
+  - Ignore the case of the matching pattern
+  - Match strings over multiple lines
+  - Treat the passed regex pattern as unicode
+  - Allow period symbols to identify new lines
+  - Match from a given index within a regex pattern
+  - Substitute the matching pattern with a supplied value
+
+{{< img src="synthetics/settings/parsing_regex_field.png" alt="Parse the response body from an HTTP Test with a regular expression" style="width:80%;">}}
```
Comment (peculiarism) [inline content/en/synthetics/settings/_index.md:75]: I'm not a huge fan of the error, I'll work on getting a new screenshot today

**Comment by peculiarism** [comment]
Looping in @AlaricCalmette and @sdeprez for their input

**Review by AlaricCalmette — APPROVED** [review]
